dart/lib/src/protocol/breadcrumb.dart

```diff
@@ -186,6 +193,7 @@ class Breadcrumb {
       if (data?.isNotEmpty ?? false) 'data': data,
       if (level != null) 'level': level!.name,
       if (type != null) 'type': type,
+      ...?unknown,
```

Comment:
This being the last item would, with the current implementation of AccessAwareMap, overwrite keys that were not read but only set. Not sure we have any like that, just saying this is technically a bug.

Reply:
I'm not sure i follow, could you elaborate further? `unknown` only has values if it was read before by `fromJson`, so it can never have known key/values in it that would be overwritten.

Reply:
Adding it before others now just to be sure.